.github/dependabot.yml

```
@@ -56,10 +56,11 @@ updates:
   - dependency-name: "@types/node"
     versions:
     - ">=15.0.0"
-    # Major version upgrades of fluentui will require extra validation
+    # Ignore fluentui/react updates until we come up with a solution for either
+    # working with enzyme or replacing it
   - dependency-name: "@fluentui/react"
     versions:
-    - ">= 9.0.0"
+    - ">= 0"
```

Just a thought, but would keeping the current version convey more specific information?
```suggestion
    - ">= 8.96.1"
```

---

Oh good call! I followed the pattern of using "0" that we're using for a couple other libraries that require manual upgrades but I don't think there's harm in using the current version number.